TradeItIosTicketSDK2/TradeItLinkedBroker.swift

```diff
@@ -51,7 +51,7 @@ import PromiseKit
                             handler(TradeItErrorResult(
                                 title: "Authentication failed",
                                 message: "The security question was canceled.",
-                                code: .brokerAuthenticationError
+                                code: .sessionError
```

Comment:
Setting it to `.sessionError` means that when `authenticateIfNeeded` is called again it will try to create a session. Before it was just showing an error.